Client/src/components/contanier/staff/Student/ViewStudent/index.css

```diff
@@ -0,0 +1,49 @@
+.view-student-titel {
+  color: #06468f;
+  font-size: 34px;
+  font-weight: bold;
+  margin-top: 50px;
+  margin-left: 39px;
+  margin-bottom: 9px;
+}
+.view-student-line {
+  width: 204px;
+  border-bottom: 1.5px solid #06468f;
+  margin-top: 100px;
+  margin-left: 39px;
+}
+.view-student {
+  width: 80%;
+  height: 641px;
+}
+
+.search-button-div {
+  display: flex;
+}
+
+.search-button-div button {
+  font-size: 14px;
+  border: none;
+  margin-top: 16px;
+  margin-left: 20px;
+  width: 180px;
+  height: 38px;
+}
+.search-button-div input {
+  margin-left: 82px;
+  margin-bottom: 35px;
+  margin-top: 15px;
```

Comment:
@naremanhilles Here also you could combine the margins into a single style: `margin: 15px 0 35px 82px;`

Reply:
i fexed it you can see :+1: